docs/content/preview/yugabyte-platform/_index.md

```diff
@@ -12,10 +12,15 @@ menu:
     weight: 10
 type: indexpage
 breadcrumbDisable: true
+body_class: yb-page-style
+resourcesIntro: Quick Links
+resources:
+  - title: What's new in YugabyteDB Anywhere
+    url: /preview/releases/release-notes/v2.15/
+  - title: YugabyteDB Anywhere FAQ
+    url: /preview/faq/yugabyte-platform/
 ---
```

Comment:
maybe put Request trial here instead with link to https://www.yugabyte.com/anywhere/

Reply:
done